docs/api.md

```diff
@@ -408,6 +409,7 @@ Updates the attributes of the current user.
 #### Response codes
 
 - 422 if any attributes are unknown (see [error: unknown attribute names](#unknown-attribute-names))
+- 403 if any attributes are unwritable (see [error: unwritable attributes](#unwritable-attributes))
```

Comment:
We'll need to update GDS API adaptors and possible frontend too right?

Makes me wonder a bit about piping all of this through frontend a bit... we've having to reproduce a lot of the error handling and replies there 🤔 especially when the api already uses problem details to send back a lot of good error data here.

Reply:
No, the existing code won't need to change.  In practice, I don't think apps will need to worry about distinguishing the two types of 403 error, unless we start turning attributes which were once writable unwritable (but we shouldn't do that without letting people who expect to be able to update those attributes know).